src/atom-environment.coffee

```diff
@@ -117,7 +117,7 @@ class AtomEnvironment extends Model
 
   # Call .loadOrCreate instead
   constructor: (params={}) ->
-    {@blobStore, @applicationDelegate, @window, @document, configDirPath, @enablePersistence} = params
+    {@blobStore, @applicationDelegate, @window, @document, configDirPath, @enablePersistence, loadBaseStylesheetsOnly} = params
```

Comment:
Maybe `onlyLoadBaseStyleSheets`? "Style sheet" should definitely be two words. After that, :ship: